### DISCUSSION TITLE
Cursor flickering in command mode

### DISCUSSION THREAD

**Discussion by azaidrahman** [folke/noice.nvim#935] [Q&A]
https://github.com/user-attachments/assets/07c3e4a7-5f0b-40a3-8b4b-c82f23d68611

I get this flickering cursor everytime I go into command mode. My package manager is Lazy and it doesnt seem to have any issues loading noice.nvim .

Noice version: 4.5.0
nvim-notify: 3.13.5
NVIM v0.10.1

Closest relevant thread I could find was : https://github.com/folke/noice.nvim/discussions/743 but this is not the same issue.

Please let me know if i can give any more relevant information.

Thanks,

**Comment by iamxiaojianzheng** [comment]
https://github.com/folke/noice.nvim/issues/931
